### PR TITLE
[#1079695] add dependency popup to Gantt component

### PR DESCRIPTION
**Pull request by PavelRubis**

### CODE REVIEW

**Review by xaylpq — APPROVED** [review]

File: src/components/Chart/DependencyPopup.vue [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+          startX = fromTask.xP;
+          stopX = toTask.xP + toTask.widthP;
+          break;
+        case 'endToStart':
+        default:
+          startX = fromTask.xP + fromTask.widthP;
+          stopX = toTask.xP;
+          break;
+      }
+      const width = parseInt(defaultStyles.width.replace('px', ''));
```
Comment (stasguryev) [inline src/components/Chart/DependencyPopup.vue]: defaultStyles.width as number

File: src/components/Chart/DependencyPopup.vue [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+  },
+  methods: {
+    /**
+     * @description Gets formated and localized duration string from the string of defined format
+     *
+     * @param {any} durationStr
+     * @returns {string} Formated and localized duration string (both formats defines by options)
+     */
+    getFormatedDuration(durationStr) {
+      const localization = this.root.state.options.locale.dependencyPopup.duration;
```
Comment (stasguryev) [inline src/components/Chart/DependencyPopup.vue]: null reference execption?

File: src/components/Chart/DependencyPopup.vue (hunk 1 above)
Comment (PavelRubis) [reply src/components/Chart/DependencyPopup.vue]: done

File: src/components/Chart/DependencyPopup.vue (hunk 2 above)
Comment (PavelRubis) [reply src/components/Chart/DependencyPopup.vue]: done

**Pull request merged**